src/community/wms-eo/src/main/java/org/geoserver/wms/eo/web/EoLayerGroupAbstractPage.java

```diff
@@ -501,14 +501,14 @@ private boolean checkDimensions(EoLayerGroupEntry entry) {
      */
     protected abstract void onSubmit(LayerGroupInfo lg);
     
-    class GroupNameValidator extends AbstractValidator {
+    class GroupNameValidator implements IValidator<String> {
 
         @Override
-        protected void onValidate(IValidatable validatable) {
-            String name = (String) validatable.getValue();
+        public void validate(IValidatable<String> iv) {
+            String name = (String) iv.getValue();
             LayerGroupInfo other = getCatalog().getLayerGroupByName(name);
             if(other != null && (layerGroupId == null || !other.getId().equals(layerGroupId))) {
-                error(validatable, "duplicateGroupNameError", Collections.singletonMap("name", name));
+                iv.error(new ValidationError("duplicateGroupNameError"));
```

Comment:
Btw, the ValidationError class seems to offer a fluent interface, adding .setVariable("name", name) after the constructor  call should do the trick